### PR TITLE
Fork query does not fork tables but instead adds default table

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

Fixes getredash/redash#3572

### CODE REVIEW

File: redash/models/__init__.py [hunk 1]
```diff
@@ -631,15 +631,18 @@ def fork(self, user):
         forked_list = ['org', 'data_source', 'latest_query_data', 'description',
                        'query_text', 'query_hash', 'options']
         kwargs = {a: getattr(self, a) for a in forked_list}
-        forked_query = Query.create(name=u'Copy of (#{}) {}'.format(self.id, self.name),
+
+        cls = type(self)
+        # Query.create will add default TABLE visualization, so use constructor to create bare copy of query
+        forked_query = cls(name=u'Copy of (#{}) {}'.format(self.id, self.name),
```
Comment (arikfr) [inline redash/models/__init__.py]:
```suggestion
        forked_query = Query(name=u'Copy of (#{}) {}'.format(self.id, self.name),
```

Why not use `Query` directly?

Comment (kravets-levko) [reply redash/models/__init__.py]: @arikfr Thanks!. Done.

**Comment by arikfr** [comment]
👍

**Comment by susodapop** [comment]
Hey @arikfr, when will this land in SaaS?

**Comment by arikfr** [comment]
@susodapop soon. Maybe today.

**Pull request merged**